cypress/integration/category/filters_and_sort_bar.js

```diff
@@ -65,27 +65,4 @@ describe("Category view - filtering and sorting", () => {
       .should("have.length", 5);
   });
 
-  it("should change order of items after changing sorting option", () => {
```

Comment:
Not sure, why is it removed?

Reply:
Test was not checking if its __working properly__ , just if products are shuffled